crates/rpc/rpc-types/src/eth/engine/payload.rs

```diff
@@ -397,14 +399,20 @@ impl PayloadStatusEnum {
 /// Various errors that can occur when validating a payload or forkchoice update.
 ///
 /// This is intended for the [PayloadStatusEnum::Invalid] variant.
-#[derive(Debug, Clone, PartialEq, Eq, thiserror::Error)]
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq, Eq, thiserror::Error)]
```

Comment:
I believe the serialize and deserialize functions need some work,
serialize should be to_string, deserialize from string (missing)

I'd like some tests for these.

Reply:
The serde Derive macros are good enough for string representation. I have added tests for these. Do you want me to implement From and Into string for this?

Reply:
see comment in test